app/views/orders/_cart_line_item.html.erb

```diff
@@ -0,0 +1,44 @@
+<hr class="my-4">
+<div class="row mb-4 d-flex justify-content-between align-items-center">
+  <div class="col-md-2 col-lg-2 col-xl-2">
+    <%= image_tag cart_line_item.meal.image, class: 'img-fluid rounded-3' if cart_line_item.meal.image.attached? %>
+  </div>
+  <div class="col-md-3 col-lg-3 col-xl-3">
+    <h6 class="text-black mb-0"><%= cart_line_item.meal.name %></h6>
+    <h6 class="text-muted"><%= cart_line_item.meal.non_veg? ? 'Non-Veg' : 'Veg' %></h6>
+  </div>
+  <div class="col-md-4 col-lg-4 col-xl-3 d-flex">
+    <button class="btn btn-link px-3"
+                        onclick="this.parentNode.querySelector('input[type=number]').stepDown()">
+      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-dash-lg" viewBox="0 0 16 16">
+        <path fill-rule="evenodd" d="M2 8a.5.5 0 0 1 .5-.5h11a.5.5 0 0 1 0 1h-11A.5.5 0 0 1 2 8Z"/>
+      </svg>
+    </button>
+    <div class="mt-1">
+      <%= form_with(model: cart_line_item, url: update_line_item_quantity_path(line_item_id: cart_line_item.id), id: "quantityForm#{cart_line_item.id}", method: :post) do |form| %>
+        <%= form.number_field :quantity, class: "form-control form-control-sm" %>
+      <% end %>
+    </div>
+    <button class="btn btn-link px-3"
+                        onclick="this.parentNode.querySelector('input[type=number]').stepUp()" >
+      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-plus-lg" viewBox="0 0 16 16">
```

Comment:
Please see if we can create these icon files separately and use the file here.

Reply:
Will do in the last PR